src/main/java/cs2103/duke/Event.java

```diff
@@ -0,0 +1,25 @@
+package cs2103.duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This class encapsulates an Event Task object, which inherits from the Task class.
+ */
+public class Event extends Task {
+    protected int index;
+    protected String at;
```

Comment:
Perhaps you could consider using a more descriptive variable name in place of `at`, so that other readers can more intuitively understand what `at` means. Maybe `eventDate` or `unparsedEventDate`?